### DISCUSSION TITLE
Class "Js" not found

### DISCUSSION THREAD

**Discussion by jcgmalave** [RVxLab/filament-colorpicker#15] [Q&A] [answered]
Hi Guys, i can move on from this error: Class "Js" not found (View: C:\Users\Administrador\laravel8ecommerce\resources\views\vendor\filament-colorpicker\colorpicker.blade.php) 

Any tips? THX

**Comment by RVxLab** [comment, marked as answer]
Update to a later version and remove your published colorpicker.blade.php. This issue was fixed in 1.2.1, see #10

**Reply by jcgmalave** [reply]
Thankssssssssssssss
